main.py

```diff
@@ -0,0 +1,7 @@
+from pathlib import Path
+
+dir_path = "/mnt/c/Users/hejaz/Downloads/Telegram Desktop"
+dir_path = Path(dir_path)
+
+for item_path in dir_path.iterdir():
+    print(item_path.is_file())
```

Comment:
Do not use print, instead use logging.

```suggestion
    logger.info(item_path.is_file())
```